pyproject.toml

```diff
@@ -1,36 +1,46 @@
-[tool.poetry]
+[project]
 name = "exasol-notebook-connector"
 version = "0.5.0"
+requires-python = ">=3.10,<4.0"
 description = "Components, tools, APIs, and configurations in order to connect Jupyter notebooks to Exasol and various other systems."
 packages = [ {include = "exasol"}, ]
-authors = [ "Christoph Kuhnke <christoph.kuhnke@exasol.com>" ]
-maintainers = [ "Christoph Kuhnke <christoph.kuhnke@exasol.com>" ]
+authors = [
+    {name="Christoph Kuhnke", email="christoph.kuhnke@exasol.com"},
+]
+maintainers= [
+    {name="Christoph Kuhnke", email="christoph.kuhnke@exasol.com"},
+]
+readme = "README.md"
+license = "MIT"
+dynamic = ["classifiers", "dependencies"]
+
+[project.urls]
+Homepage = "https://github.io/exasol/notebook-connector"
+Documentation = "https://github.io/exasol/notebook-connector"
+Source = "https://github.io/exasol/notebook-connector"
+Issues = "https://github.io/exasol/notebook-connector/issues"
+
+[tool.poetry]
+requires-poetry = ">=2.1.0"
+packages = [
+    { include = "exasol" }
+]
 classifiers = [
    "Programming Language :: Python :: 3",
    "Development Status :: 3 - Alpha",
    "Framework :: Jupyter :: JupyterLab :: 3",
    "License :: OSI Approved :: MIT License",
    "Topic :: Security :: Cryptography",
 ]
-readme = "README.md"
-license = "MIT"
-
-[tool.poetry.urls]
-"Homepage" = "https://github.io/exasol/notebook-connector"
-"Documentation" = "https://github.io/exasol/notebook-connector"
-"Source" = "https://github.io/exasol/notebook-connector"
-"Issues" = "https://github.io/exasol/notebook-connector/issues"
-
 
 [tool.poetry.dependencies]
-python = ">=3.10,<4.0"
 sqlcipher3-binary = {version = ">=0.5.0", platform = "linux"}
 sqlcipher3 = {version = ">=0.5.0", platform = "darwin"}
 sqlalchemy-exasol = "^5.0.0"
 pyexasol = ">=0.26.0, < 1"
 exasol-bucketfs = "^1.0.0"
 exasol-transformers-extension = "^2.2.1"
-transformers = {extras = ["torch"], version = "^4.36.2"}
+transformers = {extras = ["torch"], version = "^4.50.0"}
```

Comment:
Due to a vulnerability we bumped this, would we want to do a release for this? If so, is there are specific team member I should coordinate with to have them do?

Reply:
Would the CVE also go away when you remove the transformers line and keep only the transformers-extensions

Reply:
And, yes we would need to do a release. We can do it next week.

Reply:
maybe, but there's an explicit transformers usage in https://github.com/exasol/notebook-connector/blob/main/exasol/nb_connector/transformers_extension_wrapper.py#L193

Reply:
ah, I see, ok until we haven't removed that, we should keep it